documentation/tutorials/prerequisites.md

```diff
@@ -31,3 +31,81 @@ $(npm bin)/truffle test --network test
 ```
 
 These tests will take a while to finish, but if set up correctly, all tests should pass.
+
+## Keys and Networks
+
+When using UMA infrastructure, you often have to open the `truffle console` or run a script using `truffle exec`. When
+using truffle in the context of this repository, it's important to understand what types of keys you can use and what
+to specify for the `--network` argument.
+
+### Public Networks
+
+Public networks include the Ethereum mainnet and any public testnets, like Rinkeby, Kovan, or Ropsten. If you are using
+these networks, you'll generally need to know:
+
+1. Which public network you intend to use.
```

Comment:
Can we make an exhaustive list of every network we support and what they are called here? Aka is mainnet called mainnet? I don't know!

Reply:
This sounds like a question for the engineering team? FWIW, I copied this over from the "keys and networks" section of the "explainers" section on the existing docs site. (cc @mrice32 )

Reply:
Yeah, we can add a list. You can use this as a source: https://github.com/UMAprotocol/protocol/blob/master/common/PublicNetworks.js

Reply:
Great, I'll add a sentence to the end of bullet point 1 saying that "A list of UMA supported networks is available here." and link to that link you provided, Matt.